src/include/binder/copy/bound_table_scan_info.h

```diff
@@ -12,14 +12,20 @@ struct BoundTableScanSourceInfo {
     std::unique_ptr<function::TableFuncBindData> bindData;
     binder::expression_vector columns;
 
+    // the last {numWarningDataColumns} columns are for temporary internal use
+    common::column_id_t numWarningDataColumns;
```

Comment:
updated, I named it `numWarningDataColumns` since the columns store metadata used to populate warnings, not the warnings themselves so I wanted to differentiate between those two